R/shared-methods.R

```diff
@@ -178,17 +181,22 @@ DataClass <- R6::R6Class(
       initialise_dataclass(self, ...)
     },
 
-    #' @description Download raw data using the list
-    #' of supplied data_urls.
+    #' @description Download raw data from `data_urls`, stores a named list
+    #' of the `data_url` name and the corresponding raw data table in
+    #' `data$raw`
     #' @importFrom purrr map
     download = function() {
       self$data$raw <- map(self$data_urls, csv_reader,
         verbose = self$verbose
       )
     },
 
-    #' @description Dispatch cleaning methods to `clean_common` and level
-    #' specific cleaning
+    #' @description Cleans raw data (corrects format, converts column types,
+    #' etc). Works on raw data and so should be called after `download`.
+    #' Calls the specific country cleaning method (`clean_common`). For
+    #' countries with multiple levels, level specific cleaning functions which
+    #' are defined in said country are called providing they are named
```

Comment:
are then called (i.e clean common is called and then region specific cleaning not either/or)